Feature/modulo de registro

Apfarid opened the pull request:
Creación de los componentes de registro y login, funcionalidad del registro y verificaciones.

Anderson-Palacio committed:
Dependencias iniciales

Anderson-Palacio committed:
Eliminacion archivos innecesarios

Anderson-Palacio committed:
creacion y matricula del componente Register

Anderson-Palacio committed:
refactorizacion componente Register

Anderson-Palacio committed:
componente Alerta creado

Anderson-Palacio committed:
funcion de enviar y control de errores Register component

Anderson-Palacio committed:
Login component creado

Anderson-Palacio committed:
Login component creado

Anderson-Palacio committed:
Home component creado y sweetalert2 implementado

Anderson-Palacio committed:
Funcionalidad registro usuario a BE

davidauza-engineer reviewed (commented):
Hola Anderson

Excelente trabajo, muchas gracias. Quedan pendientes algunos temas que podemos dejar para la segunda entrega.

Por ahora te apoyo con la documentación y cuando hayas completado todo por favor me notificas para volver a revisar este Pull Request y hacer merge a develop.

Muchas gracias y seguimos charlando

davidauza-engineer committed:
Actualizar README.md

Se actualizó el README.md con los cambios implementados para la primera entrega.

Anderson-Palacio committed:
Card de productos agragada

Anderson-Palacio committed:
componente Sidebar creado

Anderson-Palacio committed:
Ajuste estilo componentes

Anderson-Palacio committed:
Estilos Sidebar

Anderson-Palacio committed:
Privatizacion de rutas

Anderson-Palacio committed:
Merge branch 'feature/modulo_de_registro' of https://github.com/davidauza-engineer/e-commerce-frontend into feature/modulo_de_registro

Anderson-Palacio committed:
Actualizacion Readme

davidauza-engineer reviewed (approved):
Cerrando pendientes. Saludos @Apfarid 👍
